From ddb265e3c5e99065ceeef92ecb9fd94b4c3e169e Mon Sep 17 00:00:00 2001
From: Daniel Stone <daniel@fooishbar.org>
Date: Thu, 7 May 2009 14:23:25 +0200
Subject: [PATCH 36/78] configure.ac: Add GLX_SYS_LIBS for Xvfb and Xnest

This fixes the linking for these two when using DRI2, which requires
DLOPEN_LIBS.
(cherry picked from commit ae38151ddda9984effca5bb7c582540061201dce)

Conflicts:

	configure.ac
---
 configure.ac |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

Index: xorg-server-1.5.3/configure.ac
===================================================================
--- xorg-server-1.5.3.orig/configure.ac	2008-11-05 11:46:15.000000000 -0800
+++ xorg-server-1.5.3/configure.ac	2009-09-14 19:48:11.043879237 -0700
@@ -1215,7 +1215,7 @@
 
 if test "x$XVFB" = xyes; then
 	XVFB_LIBS="$FB_LIB $FIXES_LIB $XEXT_LIB $CONFIG_LIB $DBE_LIB $XTRAP_LIB $RECORD_LIB $GLX_LIBS $RENDER_LIB $RANDR_LIB $DAMAGE_LIB $MIEXT_DAMAGE_LIB $MIEXT_SHADOW_LIB $XI_LIB $XKB_LIB $XKB_STUB_LIB $COMPOSITE_LIB $XPSTUBS_LIB"
-	XVFB_SYS_LIBS="$XVFBMODULES_LIBS"
+	XVFB_SYS_LIBS="$XVFBMODULES_LIBS $GLX_SYS_LIBS"
 	AC_SUBST([XVFB_LIBS])
 	AC_SUBST([XVFB_SYS_LIBS])
 fi
@@ -1233,7 +1233,7 @@
 
 if test "x$XNEST" = xyes; then
 	XNEST_LIBS="$FB_LIB $FIXES_LIB $MI_LIB $XEXT_LIB $DBE_LIB $XTRAP_LIB $RECORD_LIB $GLX_LIBS $RENDER_LIB $RANDR_LIB $DAMAGE_LIB $MIEXT_DAMAGE_LIB $MIEXT_SHADOW_LIB $XI_LIB $XKB_LIB $XKB_STUB_LIB $COMPOSITE_LIB $XPSTUBS_LIB $DIX_LIB $OS_LIB $CONFIG_LIB"
-	XNEST_SYS_LIBS="$XNESTMODULES_LIBS"
+	XNEST_SYS_LIBS="$XNESTMODULES_LIBS $GLX_SYS_LIBS"
 	AC_SUBST([XNEST_LIBS])
 	AC_SUBST([XNEST_SYS_LIBS])
 fi
